src/plans/plan.jl

```diff
@@ -11,8 +11,11 @@ include("quasi_newton_plan.jl")
 include("hessian_plan.jl")
 include("proximal_plan.jl")
 include("subgradient_plan.jl")
-include("particle_swarm_plan.jl")
+
 include("primal_dual_plan.jl")
+include("higher_order_primal_dual_plan.jl")
```

Comment:
Why this order? Before they were ordered alphabetically. You do require the primal dual before right? Then why not leave the particle swarm up where it was at least?